2025-05-29 09:34:52,622 INFO    Thread-2819 (_run_job):1666339 [wandb_setup.py:_flush():76] Current SDK version is 0.17.4
2025-05-29 09:34:52,622 INFO    Thread-2819 (_run_job):1666339 [wandb_setup.py:_flush():76] Configure stats pid to 1666339
2025-05-29 09:34:52,622 INFO    Thread-2819 (_run_job):1666339 [wandb_setup.py:_flush():76] Loading settings from /home/anwer/.config/wandb/settings
2025-05-29 09:34:52,622 INFO    Thread-2819 (_run_job):1666339 [wandb_setup.py:_flush():76] Loading settings from /12tb_dsk3/SEIR_model/src/wandb/settings
2025-05-29 09:34:52,622 INFO    Thread-2819 (_run_job):1666339 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'sweeps-SEIR_Skåne_Kronoberg', 'entity': 'danish2562022', 'root_dir': '/12tb_dsk3/SEIR_model/src', 'run_id': 'ebrs46vp', 'sweep_param_path': '/12tb_dsk3/SEIR_model/src/wandb/sweep-yw5aij6r/config-ebrs46vp.yaml', 'sweep_id': 'yw5aij6r'}
2025-05-29 09:34:52,622 INFO    Thread-2819 (_run_job):1666339 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2025-05-29 09:34:52,622 INFO    Thread-2819 (_run_job):1666339 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2025-05-29 09:34:52,622 INFO    Thread-2819 (_run_job):1666339 [wandb_setup.py:_flush():76] Applying login settings: {}
2025-05-29 09:34:52,622 INFO    Thread-2819 (_run_job):1666339 [wandb_init.py:_log_setup():529] Logging user logs to /12tb_dsk3/SEIR_model/src/wandb/run-20250529_093452-ebrs46vp/logs/debug.log
2025-05-29 09:34:52,623 INFO    Thread-2819 (_run_job):1666339 [wandb_init.py:_log_setup():530] Logging internal logs to /12tb_dsk3/SEIR_model/src/wandb/run-20250529_093452-ebrs46vp/logs/debug-internal.log
2025-05-29 09:34:52,623 INFO    Thread-2819 (_run_job):1666339 [wandb_init.py:_jupyter_setup():475] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7a5e81060830>
2025-05-29 09:34:52,623 INFO    Thread-2819 (_run_job):1666339 [wandb_init.py:init():569] calling init triggers
2025-05-29 09:34:52,623 INFO    Thread-2819 (_run_job):1666339 [wandb_init.py:init():576] wandb.init called with sweep_config: {'ah0': 8.639657624180614, 'ah1': 2.6415461051611615, 'bh0': 8.419661487696263, 'bh1': 8.367497757932846, 'c': 2.6972019734862194}
config: {}
2025-05-29 09:34:52,623 INFO    Thread-2819 (_run_job):1666339 [wandb_init.py:init():619] starting backend
2025-05-29 09:34:52,623 INFO    Thread-2819 (_run_job):1666339 [wandb_init.py:init():623] setting up manager
2025-05-29 09:34:52,624 INFO    Thread-2819 (_run_job):1666339 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-05-29 09:34:52,625 INFO    Thread-2819 (_run_job):1666339 [wandb_init.py:init():631] backend started and connected
2025-05-29 09:34:52,629 INFO    Thread-2819 (_run_job):1666339 [wandb_run.py:_config_callback():1382] config_cb None None {'ah0': 8.639657624180614, 'ah1': 2.6415461051611615, 'bh0': 8.419661487696263, 'bh1': 8.367497757932846, 'c': 2.6972019734862194}
2025-05-29 09:34:52,629 INFO    Thread-2819 (_run_job):1666339 [wandb_run.py:_label_probe_notebook():1334] probe notebook
2025-05-29 09:34:52,629 INFO    Thread-2819 (_run_job):1666339 [wandb_run.py:_label_probe_notebook():1344] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-05-29 09:34:52,629 INFO    Thread-2819 (_run_job):1666339 [wandb_init.py:init():720] updated telemetry
2025-05-29 09:34:52,632 INFO    Thread-2819 (_run_job):1666339 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2025-05-29 09:34:53,079 INFO    Thread-2819 (_run_job):1666339 [wandb_run.py:_on_init():2402] communicating current version
2025-05-29 09:34:53,152 INFO    Thread-2819 (_run_job):1666339 [wandb_run.py:_on_init():2411] got version response upgrade_message: "wandb version 0.19.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2025-05-29 09:34:53,152 INFO    Thread-2819 (_run_job):1666339 [wandb_init.py:init():804] starting run threads in backend
2025-05-29 09:34:53,924 INFO    Thread-2819 (_run_job):1666339 [wandb_run.py:_console_start():2380] atexit reg
2025-05-29 09:34:53,924 INFO    Thread-2819 (_run_job):1666339 [wandb_run.py:_redirect():2235] redirect: wrap_raw
2025-05-29 09:34:53,924 INFO    Thread-2819 (_run_job):1666339 [wandb_run.py:_redirect():2300] Wrapping output streams.
2025-05-29 09:34:53,924 INFO    Thread-2819 (_run_job):1666339 [wandb_run.py:_redirect():2325] Redirects installed.
2025-05-29 09:34:53,925 INFO    Thread-2819 (_run_job):1666339 [wandb_init.py:init():847] run started, returning control to user process
2025-05-29 09:34:53,986 INFO    Thread-2819 (_run_job):1666339 [wandb_run.py:_finish():2109] finishing run danish2562022/sweeps-SEIR_Skåne_Kronoberg/ebrs46vp
2025-05-29 09:34:53,986 INFO    Thread-2819 (_run_job):1666339 [jupyter.py:save_history():444] not saving jupyter history
2025-05-29 09:34:53,986 INFO    Thread-2819 (_run_job):1666339 [jupyter.py:save_ipynb():372] not saving jupyter notebook
2025-05-29 09:34:53,986 INFO    Thread-2819 (_run_job):1666339 [wandb_init.py:_jupyter_teardown():457] cleaning up jupyter logic
2025-05-29 09:34:53,986 INFO    Thread-2819 (_run_job):1666339 [wandb_run.py:_atexit_cleanup():2349] got exitcode: 0
2025-05-29 09:34:53,986 INFO    Thread-2819 (_run_job):1666339 [wandb_run.py:_restore():2332] restore
2025-05-29 09:34:53,986 INFO    Thread-2819 (_run_job):1666339 [wandb_run.py:_restore():2338] restore done
2025-05-29 09:34:57,169 INFO    Thread-2819 (_run_job):1666339 [wandb_run.py:_footer_history_summary_info():3991] rendering history
2025-05-29 09:34:57,170 INFO    Thread-2819 (_run_job):1666339 [wandb_run.py:_footer_history_summary_info():4023] rendering summary
2025-05-29 09:34:57,172 INFO    Thread-2819 (_run_job):1666339 [wandb_run.py:_footer_sync_info():3950] logging synced files
